pkg/client/shub/api.go

```diff
@@ -80,10 +79,10 @@ func getManifest(uri ShubURI, noHTTPS bool) (manifest ShubAPIResponse, err error
 	// Do the request, if status isn't success, return error
 	res, err := httpc.Do(req)
 	if res == nil {
-		return ShubAPIResponse{}, fmt.Errorf("No response received from singularity hub")
+		return ShubAPIResponse{}, fmt.Errorf("no response received from singularity hub")
```

Comment:
this is correct, but has nothing to do with the described change

Reply:
When I was changing this file while debugging, I could't help myself, I needed to fix that! But I'll change it back, this file did not need modifying.

Reply:
IMO it is perfectly fine to apply minor improvements as you work on something. See http://www.extremeprogramming.org/rules/refactor.html